uktrade/enquiry-mgmt-tool
# Update reference data to match Data Hub metadata

## Description of change

This PR updates some reference data values to match the metadata in Data Hub when an enquiry is submitted.

It also fixes a multiline string in reference data mapping in `as_utils` which was badly formatted in a previous PR.

## Test instructions

As we don't yet have integration testing in place, you can query the following Data Hub API endpoints to see the correct corresponding metadata values:  

- NewExistingInvestor: `/v4/metadata/investment-investor-type`
- InvestorInvolvement:  `/v4/metadata/investment-involvement`
- InvestmentProgramme: `/v4/metadata/investment-involvement`
